contracts/modules/assessment/Assessment.sol

```diff
@@ -189,20 +213,14 @@ contract Assessment is IAssessment, MasterAwareV2 {
     address destination,
     uint104 batchSize
   ) internal returns (uint withdrawn, uint withdrawnUntilIndex) {
-    require(
```

Comment:
why was this removed?

Reply:
its a redundant check as it duplicates the check that TokenController.mint does.

relevant discussion: https://github.com/NexusMutual/smart-contracts/pull/1201#discussion_r1701728982